### PR TITLE
Kernel version 3.0.8 Updates

### PR DESCRIPTION
**Pull request by baylessj**

### CODE REVIEW

File: v5/api/cpp/motors.rst [hunk 1]
```diff
@@ -350,6 +375,49 @@ setting ``errno``.
...
+modify_profiled_velocity
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes the output velocity for a profiled movement (`move_absolute`_ or
+`move_relative`_). This will have no effect if the motor is not following
+a profiled movement.
+
+This function uses the following values of ``errno`` when an error state is reached:
+
+- ``EINVAL``  - The given value is not within the range of V5 ports (1-21).
```
Comment (Octogonapus) [inline v5/api/cpp/motors.rst]: I hope not, since you don't specify a port here.

Comment (Octogonapus) [reply v5/api/cpp/motors.rst]: Also looks like all the other new docs do this as well.

File: v5/api/cpp/motors.rst [hunk 2]
```diff
@@ -1405,6 +1598,103 @@ setting ``errno``.
...
+**Returns:** ``1`` if the operation was successful or ``PROS_ERR`` if the operation failed,
+setting ``errno``.
+
+----
+
+set_pos_pid_full
+~~~~~~~~~~~~~~~~
+
+Sets one of ``pros::motor_pid_full_s_t`` for the motor. This intended to just modify the
+main PID constants.
```
Comment (Octogonapus) [inline v5/api/cpp/motors.rst]: I can see `set_pos_pid` as "intended to just modify the main PID constants", but isn't the point of the `_full` variants to modify everything?

File: v5/api/cpp/misc.rst [hunk 3]
```diff
@@ -297,6 +297,73 @@ This function uses the following values of ``errno`` when an error state is reac
...
+          master.set_text(0, 0, "Example");
+          pros::delay(100);
+          master.clear_line(0);
+        }
+
+
+============ ======================================================================================================
+ Parameters
+============ ======================================================================================================s
+ line         The line number at which the text will be displayed [0-2]
```
Comment (Octogonapus) [inline v5/api/cpp/misc.rst]: You aren't displaying any text here, you are clearing the line.

File: v5/api/cpp/misc.rst [hunk 4]
```diff
@@ -481,5 +610,101 @@ Analogous to `controller_get_digital_new_press <../c/misc.html#controller-get-di
...
+            pros::delay(2);
+          }
+        }
+
+
+============ ======================================================================================================
+ Parameters
+============ ======================================================================================================
+ line         The line number at which the text will be displayed [0-2]
+ col          The column number at which the text will be displayed. The width of the screen is 15 characters.
```
Comment (Octogonapus) [inline v5/api/cpp/misc.rst]: Are the columns `0` or `1` indexed?

**Comment by baylessj** [comment]
All of these points are valid, and will be addressed in the next few commits. Got the controller text setting, but missed the others.

**Pull request merged**